src/main/resources/geth/dev.json

```diff
@@ -9,6 +9,9 @@
     "byzantiumBlock": 0,
     "constantinopleBlock": 0,
     "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
```

Comment:
only berlin and london are common blocks for Geth and Besu?

Reply:
for besu I havent seen using istanbulBlock anywhere in their docs - https://besu.hyperledger.org/en/latest/public-networks/reference/genesis-items/?h=#milestone-blocks
For geth we can add 2 more which I havent included in this - https://geth.ethereum.org/docs/fundamentals/private-network#creating-genesis-block
arrowGlacier and grayGlacier

Let me update geth also to the latest blocks